components/inputs/test/input-number.visual-diff.html

```diff
@@ -7,6 +7,9 @@
 			import '../../typography/typography.js';
 			import '../input-number.js';
 			import '../../button/button-icon.js';
+			import { querySelectorComposed } from '../../../helpers/dom.js';
+
+			window.querySelectorComposed = querySelectorComposed;
```

Comment:
Apologies if `getRectTooltip`'s dependency on `querySelectorComposed` tripped you up. Ideally we'd never be poking into the internals. If this caused you any headaches, maybe we would add a log to help if the helper is `undefined`, or update the helper to dynamically import this method so we're not relying on the fixture file loading it. 🤷‍♂️

Reply:
The new vdiff stuff should hopefully resolve all of this, since the HTML & JS get merged.

Reply:
No worries! The change happened between when I'd written the code initially and when I put up the PR, so I was a bit confused why it broke, but it was a quick fix.